### ISSUE TITLE
Everything turns grey in running mode.

### ISSUE THREAD

**Issue by AllenDang** [godotengine/godot#67525]
### Godot version

4.0 Beta3

### System information

macOS v12.6

### Issue description

Here is what it looks like after running the scene.

![image](https://user-images.githubusercontent.com/18550/196135339-3205abd0-1067-42cf-89b5-ca1bb87703e5.png)

Here is what it looks like in editor.

![截屏2022-10-17 16 56 26](https://user-images.githubusercontent.com/18550/196135577-7297c833-48dd-4397-b023-b24fa8347610.png)

### Steps to reproduce

1. Create 3D scene.
2. Add camera.
3. Run.

**Comment by Zireael07** [comment]
That's because you didn't add a light to your scene. The default you see in the editor is only so that you see in editor itself.

**Comment by AllenDang** [comment]
@Zireael07 No, after add directional light and world environment, here is what it looks like
![image](https://user-images.githubusercontent.com/18550/196138332-00a5ee62-96bc-4bdf-9def-598152157264.png)

**Comment by AllenDang** [comment]
@Zireael07 Thing is, a scene looks good in beta2, will turn to total grey in beta3.

**Comment by Calinou** [comment]
Duplicate of https://github.com/godotengine/godot/issues/67449.
